Rspec/spec/.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
- [OPTIONAL] **README .md file:**
I think it would be better if you fix the  broken `[MIT](./LICENSE)` link in your README file to make it professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or an other method change this `[MIT](./LICENSE)` to  `[MIT](./MIT.md)` then the created new file in your directory name it `MIT.md` and get its content **[here](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**